modules/apps/list-type/list-type-test/src/testIntegration/java/com/liferay/list/type/service/test/ListTypeDefinitionLocalServiceTest.java

```diff
@@ -144,6 +143,42 @@ public void testDeleteListTypeDefinition() throws Exception {
 				listTypeEntry.getListTypeEntryId()));
 	}
 
+	@Test
+	public void testUpdateListTypeDefinition() throws Exception {
+		ListTypeDefinition listTypeDefinition = _addListTypeDefinition();
+
+		String listTypeDefinitionERC = RandomTestUtil.randomString();
+		String name = RandomTestUtil.randomString();
+
+		listTypeDefinition =
+			_listTypeDefinitionLocalService.updateListTypeDefinition(
+				listTypeDefinitionERC,
+				listTypeDefinition.getListTypeDefinitionId(),
+				Collections.singletonMap(LocaleUtil.getDefault(), name));
+
+		Assert.assertEquals(
+			listTypeDefinitionERC,
+			listTypeDefinition.getExternalReferenceCode());
+		Assert.assertEquals(
+			name, listTypeDefinition.getName(LocaleUtil.getDefault()));
+
+		listTypeDefinition =
+			_listTypeDefinitionLocalService.updateListTypeDefinition(
+				StringPool.BLANK, listTypeDefinition.getListTypeDefinitionId(),
+				Collections.singletonMap(LocaleUtil.getDefault(), name));
+
+		Assert.assertNotEquals(
+			listTypeDefinitionERC,
+			listTypeDefinition.getExternalReferenceCode());
```

Comment:
@murilostodolni can we check that `listTypeDefinition.getExternalReferenceCode()` is not empty?